lib/kubeclient/common.rb

```diff
@@ -202,6 +204,14 @@ def resource_name(entity_type)
         entity_type.pluralize.downcase
       end
 
+      def proxy_url_service(namespace, service, port_name)
```

Comment:
Let's try to support all the possible proxies (nodes, pods, services):

    /api/v1/proxy/nodes/{name}/{path:*}
    /api/v1/proxy/namespaces/{namespace}/pods/{name}/{path:*}
    /api/v1/proxy/namespaces/{namespace}/services/{name}/{path:*}